src/daemon/daemon-shutdown.c

```diff
@@ -139,6 +143,11 @@ static void rrdeng_flush_everything_and_wait(bool wait_flush, bool wait_collecto
 void netdata_cleanup_and_exit(int ret, const char *action, const char *action_result, const char *action_data) {
     netdata_exit = 1;
 
+#ifdef ENABLE_SYSTEMD_NOTIFY
+    // FIXME: This should be calculated dynamically instead of hard-coded.
```

Comment:
Hello @Ferroin ,

Do you plan to address this in a separate PR?

Best regards.

Reply:
It needs to be addressed in a different PR, largely because I don’t have sufficient knowledge of the internals of the agent to understand how to calculate this.

Reply:
@stelfrag, after conducting several tests, primarily on Arch Linux, I did not encounter any issues starting Netdata locally. Could you please confirm if the same applies in your environment?